spyder/widgets/editor.py

```diff
@@ -2089,12 +2091,17 @@ def set_layout_settings(self, settings):
             editorstack = splitter.widget(0)
             for index, finfo in enumerate(editorstack.data):
                 editor = finfo.editor
-                # FIXME: Temporal fix
-                try:
-                    editor.go_to_line(clines[index])
-                except IndexError:
+                # TODO: go_to_line is not working properly (the line it jumps
+                # is not the corresponding to that file). It will be fix in a

```

Comment:
Sorry, this whole line should read as

`does not correspond to the proper file). This will be fixed in a ...`